### PR TITLE
feature/modify-profile-and-create-page

### PR DESCRIPTION
**Pull request by paschal533**

### CODE REVIEW

File: Frontend/hooks/useFundraisers.ts [hunk 1]
```diff
@@ -17,12 +21,24 @@ const useFundraisers = () => {
       setIsLoadingFundraiser(false);
     };
 
+    const fetchAllFundraiserDonations = async () => {
```
Comment (dresha-dev) [inline Frontend/hooks/useFundraisers.ts]: I think it's better to create a separate hook, i.e. `useProfile` 
 or `useAccount` to get the data depending on the user account. That hook is only for getting the list of available fundraisers and it's using only for the home page.

File: Frontend/pages/profile.tsx [hunk 2]
```diff
@@ -25,6 +28,32 @@ import {
...
+
+      setTotalDonations(donations.reduce((a, b) => Number(a) + Number(b), 0).toFixed(2));
+  }
+  
+
+  useEffect(() => {
+    if(myDonations){
+      getTotalDonations();
+    }
+  }, [myDonations])
```
Comment (dresha-dev) [inline Frontend/pages/profile.tsx]: All that logic can be moved and taken from the dedicated hook or context, so once we isolate that part according to https://github.com/paschal533/FundKyiv/pull/5/files#r984349565, it should became much cleaner

File: Frontend/pages/profile.tsx [hunk 3]
```diff
@@ -63,7 +92,125 @@ const Home = () => {
...
+        <div className="flex flex-col mt-10">
+          <div className="flex flex-row w-full border-b dark:border-nft-black-1 border-nft-gray-1">
+            <p className="mb-2 text-base font-medium font-poppins dark:text-white text-nft-black-1">
+              My Donations
+            </p>
+          </div>
+          {!isLoadingUserDonations ? (
+          <div>
+            {currentAccount && (
+              <div className="items-center justify-center w-full mt-8 text-center">
```
Comment (dresha-dev) [inline Frontend/pages/profile.tsx]: To optimise performance and avoid unnecessary renders I would recommend to move all the logic and content for donations into the separate component

File: Frontend/hooks/useFundraisers.ts (hunk 1 above)
Comment (paschal533) [reply Frontend/hooks/useFundraisers.ts]: Sure, Let me work on that right away.

File: Frontend/pages/profile.tsx (hunk 2 above)
Comment (paschal533) [reply Frontend/pages/profile.tsx]: Okay, understood.

File: Frontend/pages/profile.tsx (hunk 3 above)
Comment (paschal533) [reply Frontend/pages/profile.tsx]: Sure

File: Frontend/services/api.ts [hunk 4]
```diff
@@ -64,6 +66,48 @@ export const fetchFundraisers = async (
   return items;
 };
 
+export const fetchFundraisersDetails = async (
+  limit: any,
+  offset: any,
+  currentAccount: any
```
Comment (dresha-dev) [inline Frontend/services/api.ts]:
```suggestion
  limit: number,
  offset: number,
  currentAccount: Address
```

**Pull request merged**